runtime/src/lib.rs

```diff
@@ -186,7 +183,7 @@ impl frame_system::Config for Runtime {
 }
 
 parameter_types! {
-	pub const MinimumPeriod: u64 = MINIMUM_PERIOD;
+	pub const MinimumPeriod: u64 = 1;
```

Comment:
Isn't it used by the pallet timestamp in the parachain also ?

Reply:
Yes, pallet timestamp is the only place its used now that Aura is gone. Pallet timestamp enforces that a blocks timestamp be at least this much higher than the parent blocks timestamp.

The change here is to tell pallet timestamp that block timestamps need only be at least 1ms apart.

I don't see any drawbacks. I also don't see any way to have all three of:
* the runtime enforces a minimum timestamp,
* the block timestamps reflect real wall clock time
* nodes author blocks arbitrarily fast.